src/utils.jl

```diff
@@ -108,21 +108,16 @@ function pushex!(exsigs::ExprsSigs, ex::Expr)
 end
 
 ## WatchList utilities
-function systime()
-    # It's important to use the same clock used by the filesystem
```

Comment:
8Regarding this comment - even the old `TimeVal` wasn't using the same clock as the file system, at least not on windows. Regardless, I think moving to `time()` here (which uses `TimeVal` or in the future `TimeSpec` under the hood) is more portable anyway than relying on whatever `Libc` returns.